scrapper/controllers/miner.py

```diff
@@ -42,13 +45,14 @@ def mine_user(user_name, refresh):
     '''
     user = User.query.filter(User.username==user_name).first()
```

Comment:
You mean that the invocations to a database query and a twitter request should be separated?
- Client requests a twitter user;
- API checks if user is in database:
- If not in database, return 202 and start background task;
- If in database, return 200 and json data.